docs/Installation.md

```diff
@@ -1,19 +1,28 @@
-# Docker
+# Installation
 
-Picatrix can be run as a docker image. To use it, first
+Picatrix can be installed via two ways:
+
+1. docker
+2. pip inside a virtualenv
+
+Let's explore both methods:
+
+## Docker
+
+The easiest way to install picatrix is via docker. To use it, first
 [install docker](https://docs.docker.com/engine/install/). You may also want to
 make sure you can run docker [sudo-less](https://docs.docker.com/engine/install/linux-postinstall/).
```

Comment:
running docker sudo-less is generally considered bad for security as it basically means you make whatever runs in docker root in the host machine.

Reply:
removed